deployment/README.md

```diff
@@ -257,7 +257,18 @@ Max amount of table space (GB):   "100"
 ```
 
 ### 3. Apply the production license
-Add steps of configmap update with db2adv_vpc.lic
+
+Update `RELEASE_NAME` to the namespace/project where db2 is deploy and `LICENSE_FILE` to the location of the `lic` file
+
+```
+RELEASE_NAME="db2u-cn1"
+LICENSE_FILE="db2adv_vpc.lic"
```

Comment:
Can we add in the path to emphasize path is needed? 
LICENSE_FILE="./db2adv_vpc.lic"

Reply:
I will add that.